mail_suggested_recipient_unchecked/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+#. Go to Projects and create a task linked to a customer (Joel Willis for example).
```

Comment:
This is a specific example, not a generic one, even not reproducible without installing `project`.